src/welcome/assets/components/NavigationPanel.tsx

```diff
@@ -21,7 +21,8 @@ export default class NavigationPanel extends React.Component {
       actions: [
         { name: "Configure Java Runtime", command: "java.runtime" },
         { name: "Open Java Settings", command: "workbench.action.openSettings", args: ["java."] },
-        { name: "Install Extensions...", command: "java.extGuide" }
+        { name: "Install Extensions...", command: "java.extGuide" },
+        { name: "Configure Java Formatter Settings", command: "java.formatterSettings" }
```

Comment:
Or simply `Configure Formatter Settings` as this page is dedicated for Java?

Reply:
I'm OK for both. Here I just find `Open Java Settings` also includes "Java" explicitly so I append "Java". What do you think of it?